com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/HDRenderPipeline.Prepass.cs

```diff
@@ -798,7 +799,10 @@ void RenderDBuffer(RenderGraph renderGraph, HDCamera hdCamera, TextureHandle dec
                 passData.parameters = PrepareRenderDBufferParameters(hdCamera);
                 passData.meshDecalsRendererList = builder.UseRendererList(renderGraph.CreateRendererList(PrepareMeshDecalsRendererList(cullingResults, hdCamera, use4RTs)));
                 SetupDBufferTargets(renderGraph, passData, use4RTs, ref output, builder);
-                passData.decalBuffer = builder.ReadTexture(decalBuffer);
+                if (hdCamera.frameSettings.IsEnabled(FrameSettingsField.DecalLayers))
```

Comment:
Leftovers from previous tests?
We talked about the dbuffer pass directly returning black if possible to remove this logic (and in the render pass itself)

Reply:
Yes, will change that, was planning on doing a sanity check on stuff like that later, but might as well fix this one now.